<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-4FFBNMC" failures="1" tests="9" name="Test" time="241.558" errors="0" timestamp="2024-12-26T12:01:05 IST">
  <testcase classname="testClasses.ManageProductTestClass" name="verifyProductsPageIsOpenWhileClickingOnListProducts" time="1.408">
    <skipped/>
  </testcase> <!-- verifyProductsPageIsOpenWhileClickingOnListProducts -->
  <testcase classname="testClasses.ManageProductTestClass" name="verifyProductsPageIsOpenWhileClickingOnListProducts" time="8.329"/>
  <testcase classname="testClasses.ManageProductTestClass" name="verifyToCreateA_newProduct" time="12.289"/>
  <testcase classname="testClasses.ManageProductTestClass" name="verifyToDeleteAddedProduct" time="13.205"/>
  <testcase classname="testClasses.ManageUserTestClass" name="verifyTheManageUsersPageIsOpenWhileClickingOnUsers" time="191.463">
    <skipped/>
  </testcase> <!-- verifyTheManageUsersPageIsOpenWhileClickingOnUsers -->
  <testcase classname="testClasses.ManageUserTestClass" name="verifyTheManageUsersPageIsOpenWhileClickingOnUsers" time="5.254">
    <skipped/>
  </testcase> <!-- verifyTheManageUsersPageIsOpenWhileClickingOnUsers -->
  <testcase classname="testClasses.ManageUserTestClass" name="verifyTheManageUsersPageIsOpenWhileClickingOnUsers" time="5.164">
    <skipped/>
  </testcase> <!-- verifyTheManageUsersPageIsOpenWhileClickingOnUsers -->
  <testcase classname="testClasses.ManageUserTestClass" name="verifyTheManageUsersPageIsOpenWhileClickingOnUsers" time="5.167">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testClasses.ManageUserTestClass.verifyTheManageUsersPageIsOpenWhileClickingOnUsers(ManageUserTestClass.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- verifyTheManageUsersPageIsOpenWhileClickingOnUsers -->
  <testcase classname="testClasses.ManageUserTestClass" name="verifyToAdd_aNewUser" time="12.012"/>
  <testcase classname="testClasses.BaseClass" name="@BeforeSuite createReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite createReport -->
</testsuite> <!-- Test -->
